packages/plugin-ext/src/hosted/node/plugin-reader.ts

```diff
@@ -38,13 +38,11 @@ export class HostedPluginReader implements BackendApplicationContribution {
     @optional()
     @multiInject(MetadataProcessor) private readonly metadataProcessors: MetadataProcessor[];
 
-    /**
-     * Map between a plugin's id and the local storage
-     */
-    private pluginsIdsFiles: Map<string, string> = new Map();
+    // Map between a plugin's id and the local storage
```

Comment:
/** ... */ - is a proper syntax for js-doc?

Reply:
Fixed